composer.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "rwahs/installation-profile",
+    "description": "description_text",
+    "minimum-stability": "stable",
+    "license": "proprietary",
+    "authors": [
+        {
+            "name": "Kehan Harman",
+            "email": "kehan@gaiaresources.com.au"
+        }
+    ],
+    "require-dev": {
+        "phpunit/phpunit": "5.x"
```

Comment:
It was mainly so PHPStorm would automatically phpunit as an external library so I could be lazy and get it to autocomplete. I know I could have manually added the library, but it seemed that this is something that a dependency manager should be used for.